src/wp-includes/rest-api/endpoints/class-wp-rest-block-pattern-categories-controller.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Pattern_Catergories_Controller class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ * @since      6.0.0
+ */
+
+/**
+ * Core class used to access block pattern categories via the REST API.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Block_Pattern_Categories_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructs the controller.
+	 *
+	 * @since 6.0.0
+	 */
+	public function __construct() {
+		$this->namespace = '__experimental';
```

Comment:
@ntsekouras @gziolo can this namespace be changed? If yes, should it be `wp/v2`?

Reply:
It has to. @spacedmonkey or @TimothyBJacobs would know best.

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/2488/commits/ceedc0a33b0b336e9465bf6322f4464c28f622c8.